packages/react-experiments/src/components/SelectedItemsList/SelectedItemsList.tsx

```diff
@@ -31,6 +39,13 @@ const _SelectedItemsList = <TItem extends BaseSelectedItem>(
     });
     setItems(updatedItems);
     props.onItemsRemoved?.(itemsToRemove);
+
+    if (itemsToRemove.length === 1) {
```

Comment:
I don't think there should be separate behavior for one vs multiple items, we should let the consumers decide.

Reply:
(If we do decide to keep this, it should go in UPP in the onItemsRemove function I think)